lib/debugs_bunny/tasks/trace.rake

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
```

Comment:
why do we do this in a rake task?   Why not a scheduled job instead?  What's the benefit of doing it in a rake task.  It seems disconnected in the code to me.

Are there general guidelines anywhere for what should/should not be done via rake task vs. active job?

Reply:
One reason to do it in a rake task is because it can be natively integrated with Heroku's scheduler, allowing us to automatically schedule it.
For example, we use this approach for things like KPIs:
<img width="1252" alt="Screen Shot 2020-06-01 at 10 46 17 AM" src="https://user-images.githubusercontent.com/5528713/83421125-94623300-a3f5-11ea-8d8a-b1a7801a45db.png">

Reply:
From a design perspective, another reason is that, since this is a gem, we don't want to necessarily dictate what frameworks the client is using. Rake is just one option, but a client using this gem could also wrap the functionality to perform the delete (it's just a couple lines) in whatever job framework they choose, if they want to go that route

Reply:
OK, i just worry when we have to deal with deleting stale data from multiple other tables (many others will have the same issue) that we end up having endless rake tasks configured.   I figured something like sandbox reset service would house the general cleanup of the DB on a global level where each table cleanup could be plugged in.

That may be manageable in a rake task, but it seems like it would be nice to have in the code itself.

Reply:
As for general guidelines, that's a good question. I don't have an answer to that, and I think that's a discussion that remains to be had